echopype/tests/calibrate/test_cal_params.py

```diff
@@ -2,19 +2,27 @@
 
 import numpy as np
 import xarray as xr
+from xarray.testing import assert_allclose
 
 from echopype.calibrate.cal_params import (
-    CAL_PARAMS, param2da, sanitize_user_cal_dict, _get_interp_da,
-    get_cal_params_AZFP, get_cal_params_EK, get_vend_cal_params_power
+    CAL_PARAMS,
+    param2da,
+    sanitize_user_cal_dict,
+    _get_interp_da,
+    get_cal_params_AZFP,
+    get_cal_params_EK,
+    get_vend_cal_params_power,
 )
 
+DATA = np.ones((2, 200)) * 2000
```

Comment:
```suggestion
DATA = np.random.rand(2, 200)
```

Reply:
Hi @leewujung, 
Initially, I had used the np.random.rand() as you suggested, but that caused the test assertions to fail as the test outputs also got randomized (especially for interpolation test cases). Hence I switched to np.ones().

Happy to adapt further suggestions.